lib/src/git_backend.rs

```diff
@@ -425,17 +425,6 @@ impl GitBackend {
         self.save_extra_metadata_table(mut_table, &table_lock)
     }
 
-    fn read_file_sync(&self, id: &FileId) -> BackendResult<Box<dyn Read>> {
-        let git_blob_id = validate_git_object_id(id)?;
-        let locked_repo = self.lock_git_repo();
-        let mut blob = locked_repo
-            .find_object(git_blob_id)
-            .map_err(|err| map_not_found_err(err, id))?
-            .try_into_blob()
-            .map_err(|err| to_read_object_err(err, id))?;
-        Ok(Box::new(Cursor::new(blob.take_data())))
-    }
```

Comment:
Perhaps, a better option is to make this return `Vec<u8>`? Then, `read_conflict()` can just do `serde_json::from_slice(&data)`. (sorry I didn't notice before.)